relay-dynamic-config/src/metrics.rs

```diff
@@ -41,9 +41,9 @@ impl Metrics {
 #[derive(Debug, Default, Clone, Serialize, Deserialize, PartialEq)]
 pub struct TagBlock {
     /// Name of metric of which we want to remove certain tags.
-    pub name: GlobPatterns,
+    pub name: GlobPattern,
```

Comment:
Why not use `glob2::LazyGlob`? It's already imported in this file. Do we need specific properties of `glob3`?

Reply:
simply I wasn't aware of it, I redefined to lazyglob now. about glob2 vs glob3 I need to read up on the differences here

Reply:
I did see that, but I assumed `glob2` was the old thing and it was replaced by `glob3` ...

Reply:
Here's a cheat sheet: https://github.com/getsentry/relay/issues/2426

Reply:
@jjbayer  so problem now is, lazyglob needs two asterisks to match multiple wildcards, globpatterns needs only one. It would be bad to have separate behaviour. We could create a new `GlobPatterns` type with glob2 in the background, but that'd be even more work.  
 
I'd prefer to keep the ability to send a vector of strings, it'd make the API easier. Maybe I should revert to previous with GlobPattern(GlobPatterns) ? what's your thoughts

Reply:
Yeah, OK. I would make `GlobPattern` a private helper of the `metrics` config module though (no need to introduce yet another glob utility if we want to unify them anyway).